### PR TITLE
Ob styling fixes

### PR DESCRIPTION
**Pull request by onb6**
### ℹ️ Issue

Closes [clickup ticket link](https://app.clickup.com/t/2gumtpd)
Closes [clickup ticket link](https://app.clickup.com/t/2px5q0t)
among other things

### 📝 Description

Briefly list the changes made to the code:

- Changed every instance of youth to worker (and changed some sentences so that made sense)
- Updated control explanation and survey explanation so they were good enough for demo
- Other minor word tweaks requested by Sara
- Center login page
- Styling for add new admin page (now matches figma)

### ✔️ Verification

Compare with Figma, manual testing

Provide screenshots of any new components, styling changes, or pages. 
Screen Shot 2022-04-14 at 3 22 52 PM
Screen Shot 2022-04-14 at 3 22 45 PM

### 🏕️ (Optional) Future Work / Notes

Still lots of styling fixes. Customization of explanatory text will be implemented at some point, as discussed in demo.

### CODE REVIEW

**Review by jgfino — CHANGES_REQUESTED** [review]
Looks great! Just one small typo :)

File: src/components/survey/ConfirmReviewerIdentity.tsx [hunk 1]
```diff
@@ -21,11 +21,16 @@ const ConfirmReviewerIdentity: React.FC<ConfirmReviewerIdentityProps> = ({
           <Text fontSize="lg" mb={6}>
             Welcome to JPAL&apos;s Letter of Recommendation System!
           </Text>
-          <Text mb={6}> Thanks you for taking the time to _____.</Text>
+          <Text mb={6}>
+            {' '}
```
Comment (atulyamiglani) [inline src/components/survey/ConfirmReviewerIdentity.tsx:25]: why this?

**Review by atulyamiglani — APPROVED** [review]
Looks great!

File: src/components/survey/ConfirmReviewerIdentity.tsx (hunk 1 above)
Comment (onb6) [reply src/components/survey/ConfirmReviewerIdentity.tsx:25]: oops no reason, ill remove that

File: src/components/survey/ConfirmReviewerIdentity.tsx [hunk 2]
```diff
@@ -21,11 +21,16 @@ const ConfirmReviewerIdentity: React.FC<ConfirmReviewerIdentityProps> = ({
...
+            Thanks you for taking the time to create letters of recommendation for workers that you
+            have supervised.
+          </Text>
           <Text>
-            This LoR system will Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
-            eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
-            quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.{' '}
+            This LoR system will ask you questions about people you have supervised. For each
+            wroker, your responses will be used to generate a letter of recommendation. You will
```
[the remark this answers is not in the document]
Comment (onb6) [reply src/components/survey/ConfirmReviewerIdentity.tsx:31]: ah good catch!!

**Review by rymaju — APPROVED** [review]
LGTM

**Comment by rymaju** [comment]
But a better PR title would be appreciated in the future!

**Pull request merged**